### PR TITLE
[16.0][MIG] stock_inventory_preparation_filter: Migration to 16.0

### PR DESCRIPTION
**Pull request by AlexPForgeFlow**
Migration to 16.0

Depends on: https://github.com/OCA/stock-logistics-warehouse/pull/1861

[ForgeFlow](https://github.com/ForgeFlow)

### CODE REVIEW

File: stock_inventory_preparation_filter/models/stock_inventory.py [hunk 1]
```diff
@@ -76,8 +76,8 @@ def _prepare_inventory_filter(self):
             products = Product.search(domain)
         return products
 
-    def _get_inventory_lines_values(self):
-        vals = super()._get_inventory_lines_values()
-        if self.filter == "lots":
-            vals = list(filter(lambda x: x["prod_lot_id"] in self.lot_ids.ids, vals))
-        return vals
+    # def _get_quants(self, locations):
```
Comment (mariadforgeflow) [inline stock_inventory_preparation_filter/models/stock_inventory.py]: Could you remove this part?

**Review by hspl-priva — APPROVED** [review]

**Review by hspl-hanak — APPROVED** [review]

**Comment by hspl-priva** [comment]
@AlexPForgeFlow LGTM @pedrobaeza @rousseldenis  any idea when can be merge this?

**Comment by rousseldenis** [comment]
> ###
> **[hspl-priva](/hspl-priva) ** commented [Jan 19, 2024](#issuecomment-1899934254) •

@AlexPForgeFlow Could you remove the test-requirements entry ?

**Comment by dreispt** [comment]
@AlexPForgeFlow Can you rebase, so tests are also retriggered?

**Pull request merged**